src/App.scss

```diff
@@ -1 +1,11 @@
-// styles go here
+.red {
+  color: rgb(194, 70, 70);
+}
+
+.green {
+  color: rgb(57, 141, 57);
+}
+
+.blue {
+  color: rgb(64, 64, 182);
+}
```

Comment:
Move color to variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

Reply:
Try to use HEX instead of RGB